src/plugin/electron/electronconfirmcert.js

```diff
@@ -77,31 +76,22 @@ class Controller {
   }
 
   /**
-   * Get the user-facing name for a certificate.
+   * Format the certificate expiry to a date string.
    * @param {Electron.Certificate} cert The certificate.
-   * @return {string} The name.
+   * @return {string} The formatted expiry.
    * @export
    */
-  getName(cert) {
-    return cert ? `${cert.subjectName} (${cert.issuerName})` : 'Unknown Certificate';
+  formatExpiry(cert) {
+    let expiry = 'Unknown';
+    if (cert && cert.validExpiry > 0) {
+      expiry = moment(cert.validExpiry * 1000).format('dddd, MMMM D, YYYY');
```

Comment:
Year then Month, then Day? (and skip day of week)?

Reply:
I was following the format Chrome uses when you inspect certificates. Open to suggestion, but this seems equivalent to how one would verbalize a date ("Tuesday, May 10, 2022") versus the ISO format ("2022 May 10") which is maybe more natural to read.

Reply:
My Chrome 86 doesn't show dates in the chooser. I see Subject, Issuer and Serial Number. In the inspection, I see Valid from and Valid to as numerical dates (which appear localised to DD/MM/YYYY for me).

Localised date format would be good, but that seemed like more effort that the ISO thing.

Reply:
Ahh I was looking at the certificate inspection dialog, if you look at the cert for the current page.
![Cert Dialog](https://user-images.githubusercontent.com/5685437/105921667-679f0380-5ff6-11eb-80ef-badfd2fc2e67.png)

Reply:
I see a different format again (DD-Mmm-YYYY) on a different machine. A little experimentation suggests that (at least on Windows) its whatever the "Short date" format is from the "Date & Time Settings".

Reply:
Ahh, that makes sense. Should we use `Date#toLocaleDateString` and let the browser format it? I like the full month, but I'm not sure how to format that appropriately by locale.

Reply:
For this particular ticket, OpenSphere does not support internationalization. Since `Date#toLocalDateString` is probably not what you want, I'd use a format consistent with the rest of the tool and add a ticket for potentially addressing internationalization elsewhere.

Reply:
Sounds good. I changed the format to match the timeline panel, so it will read something like `2021 Feb 1`.